docs/Developers/Newcomers_Guide.rst

```diff
@@ -313,8 +313,13 @@ where ``myfork`` is the name of your fork that you added at the previous step.
     You can also add a profile picture to your Github account so that
     you can stand out from the crowd!
 
+<<<<<<< HEAD
 Step 5. Creating a ``Pull Request``
 -----------------------------------
+=======
+Step 6. Creating a Pull Request
+-------------------------------
+>>>>>>> Newcomers_Guide.rst: Step 6 out of place bold text
```

Comment:
hey john i was wondering if this errors are caused naturally or was it my mistake somewhere? i am actually a newcomer; if this happens almost all  the time (at least for such a simple issue) then wouldnt it be cubersome while fixing other difficulty/high issues ? just curious :(